src/Compilers/CSharp/Portable/FlowAnalysis/ControlFlowPass.cs

```diff
@@ -310,6 +315,30 @@ public override BoundNode VisitLabeledStatement(BoundLabeledStatement node)
         public override BoundNode VisitGotoStatement(BoundGotoStatement node)
         {
             _labelsUsed.Add(node.Label);
+
+            // check for illegal jumps across using declarations
+            var targetLocation = node.Label.Locations[0];
```

Comment:
>targetLocation = node.Label.Locations[0]; [](start = 16, length = 41)

Would it make sense to hold on to `Start` in each case? `var targetStart = node.Label.Locations[0].SourceSpan.Start; ...`

Reply:
Agreed. Updated, makes it much more readable.